### ISSUE TITLE
RCS1163 reports False Positive for arglist

### ISSUE THREAD

**Issue by hankovich** [JosefPihrt/Roslynator#840]
**Product and Version Used**: 
Rolynator.Analyzers v3.2.0

**Steps to Reproduce**:
```csharp
public static int GetCount(__arglist)
{
    var argIterator = new ArgIterator(__arglist);
    return argIterator.GetRemainingCount();
}
```

**Actual Behavior**:
`__arglist` is marked as unused parameter.

**Expected Behavior**:
No diagnotic is reported since the parameter is indeed used.

**Comment by hankovich** [comment]
@JosefPihrt Thanks for the quick fix.

Are you sure it will also work when arglist is not the only parameter? E.g.

```csharp
public static int GetCount(int i, __arglist)
{
    var argIterator = new ArgIterator(__arglist);
    return argIterator.GetRemainingCount();
}
```
